ad hoc ipa (#2076)
saucelabs/sauce-docs: docs/test-fairy/sdk/ios/ad-hoc-ipa.md

To create an iOS App file for testing, select an archive in the **Archives organizer**.

Click the **Distribute App** button. Select an export option, and click Next.

To distribute your app to users with designated devices, choose the `Save for Ad Hoc Deployment`. The app will be code signed with the distribution certificate.

In the dialog that appears, choose a signing method and click Next.

In the distributions options screen, choose the options you prefer and click Next.

After you choose the signing options click **Next**.

In the dialog that appears, review the app, its entitlements, and the provisioning profile.

Click **Export** and finally the Finder shows the exported files. Save it to your desired location.

## Installing Your App on Test Devices Using Xcode

You can install iOS App files on devices using Xcode.

To install an app on a device using Xcode

1. Connect the device to your Mac.

2. In Xcode, choose Window > Devices and select the device under Devices.

3. In the Installed Apps table, click the Add button (+) below the table.

4. In the dialog that appears, choose the iOS App file and click Open.

> 5. The app is now added to the list of installed app end is installed n the device.

Reviewer: ```suggestion
5. The app is now added to the list of installed apps and installed in the device.
```
